android/quest/src/test/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModelTest.kt

```diff
@@ -200,10 +201,25 @@ class ProfileViewModelTest : RobolectricTest() {
     every { listActionConfig.handleClickEvent(navController, resourceData) } just runs
     profileViewModel.onEvent(event)
     every { mockProfileViewModel.onEvent(any()) } just runs
-
     verifyAll {
       mockActionConfig.interpolate(any())
       profileViewModel.onEvent(event)
     }
   }
+
+  @Test
+  fun testThatChangeManagingEntityDisplaysBottomSheetWhenEligibleEntitiesExist() {
+    val event = mockk<ProfileEvent.OverflowMenuClick>(relaxed = true)
+    val managingEntity: ManagingEntityConfig = mockk(relaxed = true)
+
+    every { event.resourceData?.baseResourceType } returns ResourceType.Group
+    coEvery { registerRepository.loadResource<Group>(any()) } returns mockk(relaxed = true)
+    every { fhirPathDataExtractor.extractValue(any(), any()) } returns "true"
+
```

Comment:
Would it be possible NOT to use mock? The FhirPathDataExtractor for instance is just a utility class for extracting data from a Resource, mocking it is unnecessary.

Reply:
Oh alright. working on it